src/main/java/pikappi/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package pikappi.command;
+
+import pikappi.Storage;
+import pikappi.TaskList;
+import pikappi.Ui;
+import pikappi.exception.PikappiException;
+
+/**
+ * Represents a command that can be executed by the user.
+ */
+public abstract class Command {
```

Comment:
I like how you have a Command class and subclasses that extend from it. You went the extra mile to make your code clearer.